app/services/request_password_reset.rb

```diff
@@ -16,15 +17,25 @@ def perform
   private
 
   def send_reset_password_instructions
+    irs_params = {
+      email: email,
+    }
+
     if Throttle.new(user: user, throttle_type: :reset_password_email).throttled_else_increment?
       analytics.throttler_rate_limit_triggered(throttle_type: :reset_password_email)
+      irs_params[:success] = true
```

Comment:
throttle triggered... sounds like unsuccessful? (we assign `true` in both branches so my guess is this is unsuccessful one)

```suggestion
      irs_params[:success] = false
```

Reply:
Good catch, typo introduced after following some Rubocop suggestions